crash-monitor.sh

```
@@ -0,0 +1,338 @@
+    if [ -d "$HOME/.vscode/logs" ]; then
+        latest_session=$(ls -1t "$HOME/.vscode/logs/" | head -1)
+        if [ -n "$latest_session" ]; then
+            echo "$HOME/.vscode/logs/$latest_session"
+        fi
+    fi
+}
+
+# Function to monitor memory usage
+monitor_memory() {
+    local log_file="$1"
+    local timestamp=$(date '+%Y-%m-%d %H:%M:%S')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)